app/views/announcements/index.html.erb

```diff
@@ -0,0 +1,119 @@
+<header>
+  HEADER
+</header>
+
+<body>
+  <div class="contianer">
+    <div class="row">
+      <div class="col-md-3">
+        <h1>Announcements</h1>
+
+          <table>
+            <tr>
+              <th>Date</th>
+              <th>Subject</th>
+              <th>Details</th>
+              <th>Updated At</th>
+            </tr>
+
+            <% @announcements.each do |a| %>
+              <tr>
+                <td><%= a.date.localtime.strftime("%D, %I:%M %p") %></td>
+                <td><%= a.subject %></td>
+                <td><%= a.details %></td>
+                <td><%= a.updated_at.localtime.strftime("%D, %I:%M %p") %></td>
+                <td><%= link_to 'Show', "#", 'data-target' => "#show_modal_#{a.id}", 'data-toggle' => 'modal' %></td>     
+                <td><%= link_to 'Edit', "#", 'data-target' => "#edit_modal_#{a.id}", 'data-toggle' => 'modal' %></td>
+                <td><%= link_to 'Delete', a, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+              </tr>
+            <% end %>
+          </table>
+
+        <p><%= link_to 'Submit New Announcement', "#", 'data-target' => "#new_modal", 'data-toggle' => 'modal'  %></p>
+
+      </div>
+      <div class="col-md-8 offset-md-1">
+          <h1>Schedule</h1>
+      </div>
+    </div>
+  </div>
+
+  <!-- Show Modal -->
```

Comment:
I think it's better if you move all the show modal into `_show.html.erb` instead of only the form. Makes this file easier to read too.